lib/agent0/agent0/hyperdrive/generate_env.py

```diff
@@ -0,0 +1,62 @@
+"""Helper script to generate a random private key."""
+import argparse
+import json
+import os
+
+import numpy as np
+from agent0.base.make_key import make_private_key
+from agent0.hyperdrive.config import get_eth_bots_config
+
+
+def generate_env(user_key: str) -> str:
```

Comment:
No need for changes here, but just so you're aware, running data stuff locally reads the .env file (e.g., getting sql credentials)

Reply:
set_env will not overwrite the existing file, only add env variables.